ClassesHomework/ClassesHomework.cs

```diff
@@ -120,9 +121,9 @@ public void Encapsulation()
             // Ill be grading this manually.
 
             Dog dog = new Dog();
-            //dog.Breed = "not able to set"; // This should not compile
-            //dog.IsSitting = false; // this should not compile
-            //dog.Age = 123; // this should not compile
+            dog.Breed = "not able to set"; // This should not compile
```

Comment:
These three lines should not compile. In order for the test to pass (or even build/run), you should need to comment these lines out.

The reason they do compile is because the properties Breed, IsSitting, and Age are all publicly settable, and this shouldn't be the case (see Goal1/Goal2/Goal3 in the comments).